### PR TITLE
proxmox_template: small refactor in get_template()

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Small improvement on the logic for determining whether a template exists.

##### ISSUE TYPE

- Refactoring Pull Request

##### COMPONENT NAME

proxmox_template

### CODE REVIEW

File: plugins/modules/proxmox_template.py [hunk 1]
```diff
@@ -145,9 +145,9 @@
 
 class ProxmoxTemplateAnsible(ProxmoxAnsible):
     def get_template(self, node, storage, content_type, template):
```
Comment (felixfontein) [inline plugins/modules/proxmox_template.py:147]: How about renaming this function to `has_template`? It's checking whether a template exists, and does not return it.

File: plugins/modules/proxmox_template.py [hunk 2]
```diff
@@ -145,9 +145,9 @@
...
+        volid = '%s:%s/%s' % (storage, content_type, template)
         try:
-            return [True for tmpl in self.proxmox_api.nodes(node).storage(storage).content.get()
-                    if tmpl['volid'] == '%s:%s/%s' % (storage, content_type, template)]
+            return any(tmpl['volid'] == volid for tmpl in self.proxmox_api.nodes(node).storage(storage).content.get())
         except Exception as e:
             self.module.fail_json(msg="Failed to retrieve template '%s:%s/%s': %s" % (storage, content_type, template, e))
```
Comment (felixfontein) [inline plugins/modules/proxmox_template.py:152]:
```suggestion
            self.module.fail_json(msg="Failed to retrieve template '%s': %s" % (volid, e))
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: plugins/modules/proxmox_template.py (hunk 1 above)
Comment (russoz) [reply plugins/modules/proxmox_template.py]: Good call! Done!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**